account_payment_order_inter_company/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module allows to map tax and account depending of the product.
```

Comment:
it looks like this is not the proper description. Meanwhile, in the module README.rst, the description looks better:
"This module allows for an inter-company billing to automatically pay the invoice in Company A as soon as the invoice in Company B is paid." So I'm not sure why these two descriptions are different, but it looks this one is not correct...

Reply:
@rvalyi , You're right, I fixed that.